guides/common/modules/con_understanding-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,24 @@
+[id="understanding_multiple_content_view_environments_{context}""]
+= Understanding multiple content view environments
+
+You can assign hosts and activation keys to one or more content view environments instead of assigning lifecycle environments and content views separately.
+A content view environment combines a specific lifecycle environment with a content view and describes which version of the content view to use.
+When you assign a host to multiple content view environments, the host gains access to the combined repositories from all its associated content view environments.
+
+The `allow_multiple_content_views` setting controls whether hosts and activation keys can be assigned to multiple content view environments.
+By default, this feature is disabled, limiting assignments to a single content view environment.
+Enabling the setting allows you to assign multiple content view environments to both hosts and activation keys.
+For more information on content settings, see {AdministeringDocURL}content_settings_{context}[Content settings] in _{AdministeringDocTitle}_.   
```

Comment:
So we don't forget: the section this links to needs a description of the allow_multiple_content_views setting. Right now the user is going to click it and find no more information. Can be in a follow-up.